client/packages/system/src/Name/ListView/ImportProperties/ImportReviewDataTable.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { FC, useState } from 'react';
+import {
+  ColumnDescription,
+  DataTable,
+  Grid,
+  NamePropertyNode,
+  NothingHere,
+  Pagination,
+  PropertyNode,
+  SearchBar,
+  TooltipTextCell,
+  useColumns,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { ImportRow } from './PropertiesImportModal';
+
+interface ImportReviewDataTableProps {
+  rows: ImportRow[];
+  properties: NamePropertyNode[] | undefined;
+}
+export const ImportReviewDataTable: FC<ImportReviewDataTableProps> = ({
+  rows,
+  properties,
+}) => {
+  const t = useTranslation();
+  const [pagination, setPagination] = useState<Pagination>({
+    page: 0,
+    first: 20,
+    offset: 0,
+  });
+  // Could filter here for only properties that are used in import
```

Comment:
Nah I think it's nice to see the blank columns for the things you left blank in the CSV!